monero-outreach-docs/translations/zh-hans/wallet_quickstart/wallet_quickstart_zh-hans.md

```diff
@@ -0,0 +1 @@

When choosing among these, keep in mind that most have an extra feature in that—the exception being the Edge Wallet—they can be configured to connect to either a third-party remote server or a server you run with a node. This is illustrated in the table.

在选择这些钱包时，请记住，他们大多数都有一个额外的特性—Edge 钱包除外—他们都可以配置为连接到第三方远程服务器或者一个您运行节点的服务器。如表所示。

## Official Wallets
官方钱包

+ [getmonero.org/downloads](https://www.getmonero.org/downloads)
If you want to run a full node, a good choice is the official Monero GUI or CLI wallet. The former provides a graphical user interface, and the latter a textual interface.

+ [getmonero.org/downloads](https://www.getmonero.org/downloads)
如果您想运行一个完整的节点，一个很好的选择是门罗币官方的GUI钱包或者CLI钱包。前者提供图示用户界面，后者提供文本界面。


## Mobile Wallets
手机钱包

+ [cakewallet.io](https://cakewallet.io/)  
Cake Wallet, a view-key-protecting light wallet, is available for iOS devices.
Cake钱包是一款用于苹果设备的查阅密钥保护型的轻钱包。


+ [monerujo.io](https://www.monerujo.io/)  
Monerujo, a view-key-protecting light wallet, is available for Android devices.
Monerujo, 是一款用于安卓设备的查阅密钥保护型轻钱包。


+ [edge.app](https://edge.app/)  
Edge Wallet is available for both Android and iOS. It is a view-key-sharing light wallet.
Edge钱包同时适用于安卓和苹果系统。是一个查阅密钥共享型轻钱包。


## Web Wallets *(Optimized for Convenience)*
Web钱包  (为便利性而优化)

+ [mymonero.com](https://mymonero.com/)  
MyMonero is a light, browser-based wallet. It is also available as an iOS app and a desktop app.
MyMonero 是一个轻量级的基于浏览器的钱包。它还可以作为iOS应用程序和桌面应用程序使用.

## Hardware Wallets *(Optimized for Security)*
硬件钱包 （为安全性而优化）

+ [ledger.com](https://shop.ledger.com/?r=92d74dc2847a)  
Ledger is a highly secure device that stores your mnemonic seed.
Ledger 是一个用于存储您的助记词的高度安全的设备。


Not Applicable
不可用

+ [trezor.io](https://trezor.io/)  
Trezor is an alternative highly secure device that stores your mnemonic seed.
Trezor是另一种高度安全的存储助记词的设备。

For a hardware wallet that provides extra security in storing and transmitting Monero even if the primary device becomes compromised, both Ledger, [ledger.com](https://shop.ledger.com/?r=92d74dc2847a), and Trezor, [trezor.io](https://trezor.io/), are well regarded. (Note that Ledger works with wallets now, while wallet support for Trezor is in progress.)

如果您需要一个即使主要设备受到破坏，在存储和传输门罗币时也能提供额外安全性的硬件钱包，推荐使用Ledger([ledger.com](https://shop.ledger.com/?r=92d74dc2847a)和Trezor 钱包([trezor.io](https://trezor.io/)。(请注意，Ledger现在与钱包已良好兼容，而Trezor的钱包支持尚需完善。)

Letting your resource and privacy preferences drive your wallet choice will maximize your Monero experience. Monero’s wealth of wallet options—from a quick MyMonero app to an authoritative full-node Monero CLI wallet combined with a Ledger or Trezor—let Monero meet its users’ diverse needs.

让您的资源和隐私偏好帮您选择钱包，将最大化您的门罗币体验。门罗币丰富的钱包选项——从快速的MyMonero应用程序到权威的全节点门罗币 CLI钱包，再加上Ledger或者trezor——让门罗币满足用户的多样化需求。

To explore further: 
更多信息

[medium.com/@anhdres/how-to-choose-a-monero-wallet-c713abb2d64d](https://medium.com/@anhdres/how-to-choose-a-monero-wallet-c713abb2d64d)

```

Comment:
The file, _wallet_quickstart-zh-hans.md_ was already merged in #132, so please remove this file too.

Reply:
Done.